accepted/0000-custom-dev-dependencies.md

```diff
@@ -0,0 +1,79 @@
+# Custom devDependencies
+
+## Summary
+
+I would like to define different sets of devDependencies, so I can install different sets of dependencies for different types of builds (unit testing / e2e / production). Aside from 
+dependencies and devDependencies you would be able to define a custom set of packages to install for development. 
+
+## Motivation
+
+I want to build two different test containers, one for Karma, one for Cypress. I am installing Cypress now when I install my devDependencies, even if 
+I want to do Karma tests. Cypress takes a long time to install, so it's really increasing my build times. Maybe I could create different versions of 
```

Comment:
Also, when I make a production build in Angular I need my Angular build tools, but I would never want to install stuff like Karma or Cypress!